src/Interface/Modules/String/ModuleTestUIDialog.cc

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Please remove all ModuleTestUI files--those are just for learning, we don't want them in the permanent codebase

Reply:
Oh yes, sorry those I forget to get rid of.